UofT-Frosh-Orientation/orientation-website
# 39 navbar component draft and questions

currently there are two navbar components, one for desktop and one for mobile, it's almost done, except for a few issues!
the storyboard also works, but the style looks a little different... 😅

**Questions:**
1. how to make underline stay when you click a navbar link?
    `:active  {} and :vistied{}` do not seem to work.... 
3. for frosh profile, when a frosh is logged in, how to connect initials to the profile icon (top right)? and how to make "Frosh Profile" button appear, and "Login" and "Register" buttons disappear?
4. how to check for mobile and desktop view, and render different components?
    tried using `react-device-detect`, with `<BrowserView></BrowserView>` and `<MobileView></MobileView>` but most likely implemented it wrong
6. how to import svg icons from fontawesome?

**To Do:**
- [ ] make the frosh profile (top right) unclickable (?), note: currently links to `/profile`
- [ ] make f!rosh logo (placeholder in top left) clickable and links to home(?)

**To Test:**
`<Navbar></Navbar>` for desktop viewing (comment or uncomment line in App.jsx), should look like below, 
![image](https://user-images.githubusercontent.com/86009011/171066331-d03a8f61-a897-444f-bc98-cfb7b4f6cdec.png)
- hover over home, about, and faq to make a purple underline appear
- hovering over buttons and profile makes buttons smaller
- clicking links should take you to the corresponding page

`<NavbarMobile></NavbarMobile>` for mobile viewing should look like below, 
![image](https://user-images.githubusercontent.com/86009011/171066234-3d1448b6-1536-4255-9702-d6f4746d8546.png)
- squares are placeholders for icons
